pbconf/pbfilter/ubuntu-16.04.pbf

```diff
@@ -0,0 +1,8 @@
+#
+# $Id$
+#
+# Filter for Ubuntu 14.04 build
+
+# for Ubuntu 14.04 use the postgres 9.3 libpq headers
+# according to thread at http://postgresql.1045698.n5.nabble.com/Details-about-libpq-cross-version-compatibility-td5723830.html the API has not changed significantly since 7.4
+filter PBBUILDDEP = postgresql-server-dev-9.4
```

Comment:
we can test it, but maybe it will be difficult to install psql 9.4 on 16.04 http://askubuntu.com/questions/765539/how-to-install-postgresql-9-4-on-ubuntu-16-04
change to 9.5?